lib/AST/ASTImporter.cpp

```diff
@@ -2793,9 +2794,8 @@ Decl *ASTNodeImporter::VisitFunctionDecl(FunctionDecl *D) {
 
   // Import the describing template function, if any.
   if (FromFT)
-    if (auto *ToFT = dyn_cast<FunctionTemplateDecl>(Importer.Import(FromFT))) {
-      ToFunction->setDescribedFunctionTemplate(ToFT);
```

Comment:
So we set this only in `VisitFunctionTemplateDecl` and this was redundant, right?

Reply:
Yes, this call is redundant.